resources/public/api/conf/2.0/application.raml

```diff
@@ -336,16 +346,41 @@ traits:
               For example, this could include ‘Declaration rejected’ or Movement Reference Number (MRN) allocated messages.
 
               Movements older than 31 days and their related messages are archived and cannot be returned by the API.
-
+              
+              The response can be returned in one of the following formats based on the provided accept header:
+        
+              * `application/vnd.hmrc.2.0+json-xml`: Json wrapped XML, where the message is provided as an XML string, sent as part of a Json payload containing other metadata
+              * `application/vnd.hmrc.2.0+json`: Json, where the message is provided as a sub document sent as part of a Json payload containing other metadata
```

Comment:
We've written Json as JSON elsewhere, so I should update this to be JSON... but I'll do this after comments.

Reply:
I agree with @drnaylor-oc and go further.

Suggestion:
 The response can be returned in one of the following formats based on the provided `Accept` header:

- `application/vnd.hmrc.2.0+json-xml`: JSON-wrapped XML, where the message is provided as an XML string sent as part of JSON payload containing other metadata
- `application/vnd.hmrc.2.0+json`: JSON, where the message is provided as a sub-document sent as part of a JSON payload containing other metadata